CosNet.API/Services/CosplayItemService.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using CosNet.API.Data.Repositories;
+using CosNet.API.Entities;
+using CosNet.API.Exceptions.Web;
+using CosNet.Shared.DTOs.CosplayItem;
+
+namespace CosNet.API.Services
+{
+    public class CosplayItemService : ICosplayItemService
+    {
+        private readonly IMapper _mapper;
+        private readonly ICosplayItemRepository _cosplayItemRepository;
+
+        public CosplayItemService(IMapper mapper, ICosplayItemRepository cosplayItemRepository)
+        {
+            _mapper = mapper;
+            _cosplayItemRepository = cosplayItemRepository;
+        }
+
+        public IEnumerable<CosplayItemDTO> GetCosplayItems(Guid cosplayId)
+        {
+            var cosplayItems = _cosplayItemRepository.GetCosplayItems(cosplayId);
```

Comment:
There is no check for if the cosplay even exists.
Requesting items for a cosplay that does not exist should throw a not found exception.